src/test/java/software/amazon/cloudformation/WrapperTest.java

```diff
@@ -467,9 +514,7 @@ public void reInvokeHandler_InProgress_returnsInProgress(final String requestDat
             verifyNoMoreInteractions(providerMetricsPublisher);
 
             // verify that model validation occurred for CREATE/UPDATE
-            if (action == Action.CREATE || action == Action.UPDATE) {
-                verify(validator).validateObject(any(JSONObject.class), any(JSONObject.class));
-            }
+            verify(validator).validateObject(any(JSONObject.class), any(JSONObject.class));
```

Comment:
Isn't the verify is `verify(validator, never())....` to make validation?    That's all the fix about.

Reply:
I put verify here because payload has no callback context - `"create.without-callback-context.request.json,CREATE", "update.without-callback-context.request.json,UPDATE"`

Reply:
Got it. the input comes from `create.without-callback-context.request.json`, i thought it's `pe` from this test.